### PR TITLE
Merge titlebar

### PR DESCRIPTION
**Pull request by Zxnii**
Merge titlebar into the window (lots of commits I know, I kept breaking things)

Windows:
image
MacOS:
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "ninetails",
   "productName": "Ninetails",
-  "version": "3.8.0",
+  "version": "3.9.0",
```
Comment (MystPi) [inline package.json:4]: Thanks for bumping the version; I often forget. 🤣

File: src/index.html [hunk 2]
```diff
@@ -1,132 +1,201 @@
 <!DOCTYPE html>
```
Comment (MystPi) [inline src/index.html:1]: I don't like the formatting, could you revert it?

**Comment by MystPi** [comment]
I'm not so sure about this... I'll think about it.

File: src/index.html (hunk 2 above)
Comment (Zxnii) [reply src/index.html:1]: sure can, i forgot i had my code formatter on so i can revert it

**Comment by webdev03** [comment]
What about Linux and desktop environment things?

**Comment by Zxnii** [comment]
yeah its disabled for linux, in future i think it'd be cool to hook into gtk and implement a custom titlebar

**Pull request closed without merging**